3.0-RELEASE-NOTES.md

```diff
@@ -5,3 +5,14 @@ always describe the impact on users and instructions for upgrading
 applications from 2.x to 3.0.
 
 See also https://github.com/strongloop/loopback/blob/master/3.0-DEVELOPING.md
+
+
+## API call for remote method has been changed.
+
+Originally, we have SharedClass.prototype.find method for looking up remote
+methods in 2.x. This is now deprecated in 3.0 and is replaced by 
+SharedClass.prototype.findMethod where you can pass in a static method or a
+prototype method name.
```

Comment:
IMO the description is not detailed enough for readers to understand how to use the new API. Please add an examples of how to use it - one example for a static method and another for a prototype method.